hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/EstimatePrecompileFeature.java

```diff
@@ -574,16 +661,14 @@ public void associateFungibleKycUnfrozenTokenWithReceiverAccount() {
 
     @Then("I call estimateGas with dissociateTokens function for fungible tokens")
     public void dissociateTokensEstimateGas() {
-        ByteBuffer encodedFunctionCall = getFunctionFromEstimateArtifact(ContractMethods.DISSOCIATE_TOKENS)
-                .encodeCallWithArgs(
-                        asAddress(receiverAccountAlias),
-                        asAddressArray(Arrays.asList(
-                                fungibleTokenId.toSolidityAddress(), fungibleKycUnfrozenTokenId.toSolidityAddress())));
+        var data = encodeData(
+                ESTIMATE_PRECOMPILE_TEST_CONTRACT,
```

Comment:
All contracts in `ContractResource` are test contracts, so it's redundant to append `_TEST_CONTRACT`. If we remove unnecessary suffix perhaps some of these multi-line `encodeData()` can revert to single line.

```suggestion
                ESTIMATE_PRECOMPILE,
```

Reply:
Done. Remove the "_TEST_CONTRACT" suffix.